### PR TITLE
[CDAP-11598] Fixes active connection in dataprep connections side panel

### PR DESCRIPTION
**Pull request by ajainarayanan**
JIRA: https://issues.cask.co/browse/CDAP-11598
Build: https://builds.cask.co/browse/CDAP-UDUT176

### CODE REVIEW

File: cdap-ui/app/cdap/components/DataPrepConnections/DataPrepConnections.scss [hunk 1]
```diff
@@ -69,12 +70,13 @@ $info-font-color: $grey-03;
         }
 
         &.expandable-menu {
-          padding: 10px 15px 10px 8px;
+          padding: 10px 0 10px 0;
```
Comment (elfenheart) [inline cdap-ui/app/cdap/components/DataPrepConnections/DataPrepConnections.scss]: nit: `10px 0`

**Review by elfenheart — COMMENTED** [review]
minor comment, otherwise LGTM

**Review by elfenheart — APPROVED** [review]
👍 LGTM

**Comment by ajainarayanan** [comment]
Latest green build: https://builds.cask.co/browse/CDAP-UDUT176-2

**Pull request merged**